etc/conda-install.sh

```diff
@@ -114,6 +115,7 @@ elif [ -n "${CHANGED// }" ]; then
 fi
 
 if [ "$RECREATE_CONDA_ENV" -eq "1" ]; then
+    conda install -n base mamba --force-reinstall
```

Comment:
@charlesbluca I think this line can be removed if we use mambaforge. Does that sound right to you?

Reply:
Yeah we can remove that for now - if we encounter unsatisfiable errors in the future relating to mamba we can try to add it back in to see if it resolves